hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -1277,6 +1279,18 @@ public static <T extends Record, R extends GeneratedMessageV3> R pbjToProto(
         }
     }
 
+    public static @NonNull CustomFee fromFcCustomFee(@Nullable final FcCustomFee fcFee) {
+        try (final var bais =
```

Comment:
Was there a reason you didn't use,
```
    return protoToPbj(Objects.requireNonNull(fcFee).asGrpc(), CustomFee.class);
```
?

Reply:
`FcCustomFee` is not protobuf type right ?

Reply:
Oh probably did not realize this method. I will use it.